src/DynamoCore/Extensions/ReadyParams.cs

```diff
@@ -38,6 +38,19 @@ public IWorkspaceModel CurrentWorkspaceModel
             }
         }
 
+        private ICommandExecutive commandExecutive;
+        public virtual ICommandExecutive CommandExecutive 
```

Comment:
The reason why it's virtual is so that it could be overridden by `ViewModelParams` which could decide to instantiate it's own version if `ExtensionCommandExecutive` that took in `DynamoViewModel` as parameter and thus call the `ExecuteCommand` version on `DynamoViewModel`.